shell/initialize/client.js

```diff
@@ -247,20 +245,6 @@ async function render(to, from, next) {
   }
 }
 
-// Fix components format in matched, it's due to code-splitting of vue-router
-function normalizeComponents(to, ___) {
-  flatMapComponents(to, (Component, _, match, key) => {
-    if (typeof Component === 'object' && !Component.options) {
```

Comment:
When investigating it appears that all component we used had the options object which caused this to be a noop.